dockerfiles/go-1.22.Dockerfile

```
@@ -1,18 +1,16 @@
 ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="go.mod,go.sum"
 
-# By default, Go uses $HOME for GOCACHE. We need to override $HOME in some tests,
-# so let's ensure Go uses a different directory for caching.
-RUN mkdir -p /cache
-ENV GOCACHE=/cache
-
 WORKDIR /app
 
-COPY go.mod go.sum ./
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)